inc/integrations/host-providers/class-bunnynet-host-provider.php

```
@@ -0,0 +1,454 @@
+
+		$zone_id = defined('WU_BUNNYNET_ZONE_ID') && WU_BUNNYNET_ZONE_ID ? WU_BUNNYNET_ZONE_ID : '';
+
+		if (! $zone_id) {
+			return;
+		}
+
+		if (! str_contains($subdomain, (string) $current_site->domain)) {
+			return; // Not a sub-domain of the main domain.
+		}
+
+		$original_subdomain = $subdomain;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$original_subdomain'. (undefined)

(UnusedLocalVariable)